ml_ops.sh

```diff
@@ -74,13 +74,13 @@ hdfs dfs -rm -R -f ${HDFS_SCORED_CONNECTS}
 
 # Add -p <command> to execute pre MPI command.
 # Pre MPI command can be configured in /etc/duxbay.conf
-# In this script, after the line after --mpicmd ${MPI_CMD} add:
-# --mpiprep ${MPI_PREP_CMD}
+# In this script, after the line after -c ${MPI_CMD} add:
```

Comment:
I like using the verbose names better in the comment...

actually,  what does it hurt to add

--mpiprep ${MPI_PREP_CMD}

since a properly written conf file will have an empty command there ?  or does the spark options handler not allow empty strings arguments (i think it does, I mean, it _should_)